plugin/gentoo-common.vim

```diff
@@ -56,8 +56,19 @@ fun! GentooGetPythonTargets()
     if exists("g:gentoopythontargets") && g:gentoopythontargets != ""
         return g:gentoopythontargets
     else
-        let l:py3 = system("python -c 'import epython; print(epython.EPYTHON)'")
-        let l:py3 = substitute(l:py3, "\n", "", "g")
+        let l:py3_list = []
+        for item in readfile('/etc/python-exec/python-exec.conf')
+            let l:py = matchstr(item,'^python.*')
```

Comment:
This would skip pypy3. You should include all lines, except for empty lines, comments (`#`) and disables (`-`).